.dockerignore

```diff
@@ -1,22 +1,39 @@
 .git
 .github
 .vscode
-
-.dockerignore
 .gitignore
 .golangci.yml
-*.out
 bin/
-hack/.bin/
-hack/tools/bin/
+**/*.yaml
+hack/
 out/
 docs/
-scripts/
+packaging/
+templates/
 **/*.md
-*.test
-cluster-api-provider-vsphere
-examples/provider-components/provider-components*.yaml
-test/
+tilt-settings.json
+tilt-settings.yaml
+tilt.d/
+Tiltfile
```

Comment:
There is no titlfile in this repo (probably applies to most of the tilt files here)